languages/elixir/exercises/concept/list-comprehensions/.meta/example.ex

```diff
@@ -0,0 +1,15 @@
+defmodule BoutiqueSuggestions do
+  def get_combinations(tops, bottoms, options \\ []) do
+    maximum_price = Keyword.get(options, :maximum_price, 100.00)
```

Comment:
Could maximum price be an argument directly instead inside options? Normally it makes perfect sense to have options in such a context, but since we're teaching list comprehensions here, I wouldn't distract students with keyword lists.

Reply:
I could see what it's like with that instead.  I wanted to add in options like this because using the maximum price won't be the first step in the instructions, so I didn't want it to distract from the progression of the exercise thinking of it from a TDD approach. When added in like this they can add the blank options, then add the option in for maximum_price when they get to that step.

Reply:
That makes sense! Couldn't really know that without seeing the instructions first 👀